Add static page for IPT

SamuelAlber opened the pull request:
Also including pictures of past IPT competitions

SamuelAlber committed:
First version of the static IPT page

Also including pictures of past IPT competitions

SamuelAlber committed:
Update ipt.html

carterturn reviewed (changes requested):
This is pretty good. There are a couple of content aspects I think should be addressed. Also, the latest version of `dev-stable` needs to be merged in before this can be merged into `dev-stable`.

Currently the teams list is from 2018-2019, while the paragraph describes our participation in 2019-2020. I would recommend either removing the teams list or adding the years that various teams were active.

This was a mistake on my part, but the target audience for the current text is a bit different than the target audience for the webpage. I would remove the sentence "Efforts such as this closely follow the mission of the national SPS chapter." I would also add another paragraph saying something along lines of "opportunities to participate in the next IPT will be announced soon."

Finally, I think having a slightly longer line length might be nice. In particular, I would not split the <img> tags on lines 101 and 104. This is not as important though.

carterturn committed:
Merge branch 'dev-stable' into static-ipt

SamuelAlber committed:
Update IPT.html with suggestions

Removed teams section, lengthened the line length, included paragraph about new updates for the 2020-21 IPT, and removed sentence about the national SPS chapter.

carterturn reviewed (changes requested):
This looks good!
One small change: since it is no longer referenced, could you delete the extra styling?

SamuelAlber committed:
Removed extra styling

SamuelAlber committed:
Changed paragraph text style to "content"

carterturn committed:
Merge branch 'dev-stable' into static-ipt

carterturn reviewed (approved):
This looks good!
